daemon/src/model/cfd.rs

```diff
@@ -557,6 +557,15 @@ impl Cfd {
             .map(|dlc| dlc.settlement_event_id.timestamp)
     }
 
+    fn margin(&self, position: Position) -> Amount {
+        match self.position {
```

Comment:
Bug!

You need to use `position` here, not `self.position`.

Reply:
hah, yes! fixed locally when adding test :)

Reply:
ah, it was already fixed, just accidentally squashed into the other commit! should have used `git absorb` :)

Reply:
(it went away anyway with the other suggestion)